bookshelf/google-checks.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0"?>
```

Comment:
google-checks can definitely go in the root and be referenced from the children.